aws/policy/paas.yaml

```diff
@@ -24,6 +24,14 @@ Statement:
   - Sid: AllowResourceRestrictedActionsWhichIncurNoFees
     Effect: Allow
     Action:
+      - cloudfront:CreateDistribution
+      - cloudfront:CreateStreamingDistribution
```

Comment:
The create/update actions should always be under a Sid that has "ActionsWhichIncurFees" in the name.  Right now everything is in an "ActionsWhichIncurNoFees" Sid and only separated by whether they're resource restricted, the relevant actions should be moved.

For whatever reason, `create_distribution` only works when I remove the resource restriction (despite what the docs suggest).  IMO this is fine, so let's remove the resource restriction on the create actions (by moving these actions to an appropriate Sid).

Reply:
updated